img/messages/legal-message.svg

```diff
@@ -0,0 +1,3 @@
+</svg>
```

Comment:
In order to avoid duplication, could you use the already existing `img/features/mvl/legalMessage.svg`?

Reply:
Totally missed it 🤭  thanks!
Here you go https://github.com/pagopa/io-app/pull/3599/commits/013f02522f9a7bce8f44ee89f4a5eca50e1ae8e5